listenbrainz/webserver/views/stats_api.py

```diff
@@ -483,14 +483,377 @@ def get_artist_activity(user_name: str):
     :statuscode 404: User not found
     :resheader Content-Type: *application/json*
     """
-    user, stats_range = _validate_stats_user_params(user_name)
-    offset = get_non_negative_param("offset", default=0)
-    count = get_non_negative_param("count", default=DEFAULT_ITEMS_PER_GET)
-    stats = db_stats.get(user["id"], "release_groups", stats_range, EntityRecord)
-    if stats is None:
-        raise APINoContent('')
-
-    release_groups_list, _ = _process_user_entity(stats, offset, count, entire_range=True)
+    # user, stats_range = _validate_stats_user_params(user_name)
+    # offset = get_non_negative_param("offset", default=0)
+    # count = get_non_negative_param("count", default=DEFAULT_ITEMS_PER_GET)
+    # stats = db_stats.get(user["id"], "release_groups", stats_range, EntityRecord)
+    # if stats is None:
+    #     raise APINoContent('')
+
+    # release_groups_list, _ = _process_user_entity(stats, offset, count, entire_range=True)
+    temp_data = {
```

Comment:
This should not have been committed. 
It's totally fine to do on your machine, but please check every change that you commit before pressing the button.